chain-vote/src/cryptography/zkps/mod.rs

```diff
@@ -1,9 +1,12 @@
-// Correct encryption will be used once we proceed with DKG protocol
+// CorrectElGamalDecrZkp used for disclosing randomness by committee members
 // mod correct_decryption;
+// CorrectHybridDecrKeyZkp used for DKG algorithm
+// mod correct_hybrid_decryption_key;
```

Comment:
why is this commented?

Reply:
linter complains that the code is not being used